forem/forem
# Try new placement area for display ads, post_sidebar

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

* Closes https://github.com/forem/forem/issues/18638

## Description

We want to have a new placement area option for Display Ad: in the right sidebar, below author & series information, on an individual `article#show` page.

## QA Instructions, Screenshots, Recordings

* Create a new display ad
* Choose `Sidebar Right (Individual Post)` for it's Placement Area
* Save the ad
* View an article
* See the ad

<img width="1291" alt="Screen Shot 2022-11-04 at 15 30 16" src="https://user-images.githubusercontent.com/2077/200000181-fd8beed3-851a-4d54-869e-00cb00dce149.png">

## Added/updated tests?

- [x] No, and this is why: existing tests should cover this option? unless we've under-tested views?

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

Might want to monitor the caching of this?

## [optional] What gif best describes this PR or how it makes you feel?

![stranger ad](https://media.giphy.com/media/l1KcRlOCAMa5A7jnW/giphy.gif)
